airbyte-integrations/connectors/source-gnews/source_gnews/gnews.yaml

```diff
@@ -44,7 +146,12 @@ definitions:
     nullable: "{{ ','.join(config['nullable']) }}"
     from: "{{ stream_slice['start_time'] }}"
     to: "{{ stream_slice['end_time'] }}"
+  schema_loader:
+    type: StaticSchemaLoader
```

Comment:
This should be `InlineSchemaLoader` correct?

Reply:
😬 yep! Fixed.